### ISSUE TITLE
Error 405 while running reportportal

### ISSUE THREAD

**Issue by mchundi** [reportportal/reportportal#1993]
Hi,
I am using version 5.7.4. I see the below errors while running my tests:
```
[main] WARN com.epam.reportportal.utils.properties.PropertiesLoader - Duplicate key found in property overrides.
12:40:45 WARNING: An illegal reflective access operation has occurred
12:40:45 WARNING: Illegal reflective access by retrofit2.Platform to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
12:40:45 WARNING: Please consider reporting this to the maintainers of retrofit2.Platform
12:40:45 WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
12:40:45 WARNING: All illegal access operations will be denied in a future release
[main] INFO com.epam.reportportal.service.Launch - Rerun: false
[main] WARN com.epam.reportportal.service.statistics.StatisticsService - Unable to complete execution of all dependencies
java.io.IOException: Unexpected response code for CONNECT: 405
	at okhttp3.internal.connection.RealConnection.createTunnel(RealConnection.java:427)
	at okhttp3.internal.connection.RealConnection.connectTunnel(RealConnection.java:236)
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:177)
	at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.java:224)
	at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.java:108)
	at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.java:88)
	at okhttp3.internal.connection.Transmitter.newExchange(Transmitter.java:169)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:41)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229)
	at okhttp3.RealCall.execute(RealCall.java:81)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:204)
	at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:46)
	at io.reactivex.Observable.subscribe(Observable.java:12267)
	at io.reactivex.internal.operators.observable.ObservableSubscribeOn$SubscribeTask.run(ObservableSubscribeOn.java:96)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1589)
```

All the services are up and running and are healthy. I see the below error in logs:
POSTGRESS ERROR LOGS:

```
2023-04-27 19:04:15.944 UTC [1398] DETAIL:  Process 1398 waits for ShareLock on transaction 101323840; blocked by process 1369.
	Process 1369 waits for ShareLock on transaction 101323852; blocked by process 1370.
	Process 1370 waits for ShareLock on transaction 101323841; blocked by process 1398.
	Process 1398: SELECT t.item_id FROM test_item t WHERE t.unique_id = $1 AND t.launch_id = $2  AND t.parent_id = $3 AND t.has_stats AND t.retry_of IS NULL ORDER BY t.start_time DESC, t.item_id DESC LIMIT 1 FOR UPDATE
	Process 1369: insert into public.issue (auto_analyzed, ignore_analyzer, issue_description, issue_type, issue_id) values ($1, $2, $3, $4, $5)
	Process 1370: insert into public.issue (auto_analyzed, ignore_analyzer, issue_description, issue_type, issue_id) values ($1, $2, $3, $4, $5)
2023-04-27 19:04:15.944 UTC [1398] HINT:  See server log for query details.
2023-04-27 19:04:15.944 UTC [1398] CONTEXT:  while locking tuple (1113139,2) in relation "test_item"
2023-04-27 19:04:15.944 UTC [1398] STATEMENT:  SELECT t.item_id FROM test_item t WHERE t.unique_id = $1 AND t.launch_id = $2  AND t.parent_id = $3 AND t.has_stats AND t.retry_of IS NULL ORDER BY t.start_time DESC, t.item_id DESC LIMIT 1 FOR UPDATE
```
SERVICE API ERROR LOGS:
```
2023-04-27 18:20:13.221 ERROR 1 --- [io-8585-exec-33] .AuthUtils$SerialUidReplacingInputStream : Potentially Fatal Deserialization Operation.

java.io.InvalidClassException: Overriding serialized class version mismatch: local serialVersionUID = 550 stream serialVersionUID = 520
	at com.epam.ta.reportportal.auth.util.AuthUtils$SerialUidReplacingInputStream.readClassDescriptor(AuthUtils.java:105)
	at java.base/java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1992)
	at java.base/java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1870)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2201)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:489)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:447)
	at java.base/java.util.ArrayList.readObject(ArrayList.java:929)
	at java.base/jdk.internal.reflect.GeneratedMethodAccessor189.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1046)
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2357)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496)
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496)
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:489)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:447)
	at com.epam.ta.reportportal.auth.util.AuthUtils.deserializeSafely(AuthUtils.java:69)
	at com.epam.ta.reportportal.auth.CombinedTokenStore.readAuthentication(CombinedTokenStore.java:68)
	at org.springframework.security.oauth2.provider.token.store.JwtTokenStore.readAuthentication(JwtTokenStore.java:70)
	at com.epam.ta.reportportal.auth.CombinedTokenStore.readAuthentication(CombinedTokenStore.java:55)
	at com.epam.ta.reportportal.auth.CombinedTokenStore$$FastClassBySpringCGLIB$$753697c.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:783)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:753)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:123)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:388)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:753)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:698)
	at com.epam.ta.reportportal.auth.CombinedTokenStore$$EnhancerBySpringCGLIB$$7c24aad6.readAuthentication(<generated>)
	at org.springframework.security.oauth2.provider.token.DefaultTokenServices.loadAuthentication(DefaultTokenServices.java:242)
	at org.springframework.security.oauth2.provider.token.DefaultTokenServices$$FastClassBySpringCGLIB$$5a1f25c.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:689)
	at org.springframework.security.oauth2.provider.token.DefaultTokenServices$$EnhancerBySpringCGLIB$$a07cd176.loadAuthentication(<generated>)
	at org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationManager.authenticate(OAuth2AuthenticationManager.java:87)
	at org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationProcessingFilter.doFilter(OAuth2AuthenticationProcessingFilter.java:154)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:103)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:89)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:110)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:80)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:55)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:211)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:183)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:354)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:267)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.doFilterInternal(WebMvcMetricsFilter.java:96)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:829)
```
Can you please help resolve the error?

**Comment by janmekota** [comment]
Hello, I noticed the error in the API service also in 5.7.3.
`java.io.InvalidClassException: Overriding serialized class version mismatch: local serialVersionUID = 550 stream serialVersionUID = 520`

**Comment by PIDlooper** [comment]
Hello @mchundi , can you please provide some more details regarding the deployment and also identify the platform OS that you are running RP 5.7.4 on?

**Comment by mchundi** [comment]
OS version: 
```
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
```
RP is installed using docker.
reportportal/service-auto-analyzer:5.7.4             
reportportal/service-auto-analyzer:5.7.4             
reportportal/service-api:5.7.4                       
reportportal/service-jobs:5.7.4                      
reportportal/service-authorization:5.7.4             
reportportal/service-metrics-gatherer:5.7.3          
reportportal/service-index:5.7.3                     
rabbitmq:3.7.16-management                           
traefik:v2.0.7                                       
docker.elastic.co/elasticsearch/elasticsearch:7.10.1 
minio/minio:RELEASE.2020-10-27T04-03-55Z             
postgres:12-alpine                                   
reportportal/service-ui:5.7.4                        

I am using RP testNG agent 5.1.4

**Comment by irynakozak2** [comment]
@mchundi Hi! Unfortunately, the Deserialization Error in the service-API logs are a known issue since v5.7.3, but there is no impact on the ReportPortal work. We assume to resolve it in further versions. 

Regarding the error `[main] WARN com.epam.reportportal.service.statistics.StatisticsService - Unable to complete execution of all dependencies
java.io.IOException: Unexpected response code for CONNECT: 405`: if all services are healthy, possibly it could be the issue on the agent side. Could you please bump your RP testNG version to [5.2.0](https://mvnrepository.com/artifact/com.epam.reportportal/agent-java-testng/5.2.0) and check if the issue is still relevant?

If the issue still persists after the agent update, please share the full service-API log for further investigation.

Waiting for your reply.

**Comment by irynakozak2** [comment]
@mchundi Hi! Long time no hear from you, so I close that issue for now. If that issue is still relevant for you after the upgrade - please reply to this ticket or consider creating a new one.
Thank you!

**Comment by irynakozak2** [comment]
@mchundi Hi! Long time no hear from you, so I close that issue for now. If that issue is still relevant for you after the upgrade - please reply to this ticket or consider creating a new one.
Thank you!

**Comment by mchundi** [comment]
@irynakozak2 I have upgraded TestNG agent version to 5.3.1 and I still see 405 error.
Here is the service-API log
`2023-08-24 18:54:23.977 ERROR 1 --- [585-exec-160036] .AuthUtils$SerialUidReplacingInputStream : Potentially Fatal Deserialization Operation.

java.io.InvalidClassException: Overriding serialized class version mismatch: local serialVersionUID = 550 stream serialVersionUID = 520
	at com.epam.ta.reportportal.auth.util.AuthUtils$SerialUidReplacingInputStream.readClassDescriptor(AuthUtils.java:105)
	at java.base/java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1992)
	at java.base/java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1870)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2201)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:489)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:447)
	at java.base/java.util.TreeSet.readObject(TreeSet.java:524)
	at java.base/jdk.internal.reflect.GeneratedMethodAccessor2304.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1046)
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2357)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496)
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496)
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496)
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496)
	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:489)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:447)
	at com.epam.ta.reportportal.auth.util.AuthUtils.deserializeSafely(AuthUtils.java:69)
	at com.epam.ta.reportportal.auth.CombinedTokenStore.readAuthentication(CombinedTokenStore.java:68)
	at org.springframework.security.oauth2.provider.token.store.JwtTokenStore.readAuthentication(JwtTokenStore.java:70)
	at com.epam.ta.reportportal.auth.CombinedTokenStore.readAuthentication(CombinedTokenStore.java:55)
	at com.epam.ta.reportportal.auth.CombinedTokenStore$$FastClassBySpringCGLIB$$753697c.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:783)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:753)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:123)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:388)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:753)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:698)
	at com.epam.ta.reportportal.auth.CombinedTokenStore$$EnhancerBySpringCGLIB$$7c24aad6.readAuthentication(<generated>)
	at org.springframework.security.oauth2.provider.token.DefaultTokenServices.loadAuthentication(DefaultTokenServices.java:242)
	at org.springframework.security.oauth2.provider.token.DefaultTokenServices$$FastClassBySpringCGLIB$$5a1f25c.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:689)
	at org.springframework.security.oauth2.provider.token.DefaultTokenServices$$EnhancerBySpringCGLIB$$a07cd176.loadAuthentication(<generated>)
	at org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationManager.authenticate(OAuth2AuthenticationManager.java:87)
	at org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationProcessingFilter.doFilter(OAuth2AuthenticationProcessingFilter.java:154)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:103)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:89)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:110)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:80)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:55)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:211)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:183)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:354)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:267)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.doFilterInternal(WebMvcMetricsFilter.java:96)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:829)
`

run log
`[main] INFO com.epam.reportportal.service.Launch - Rerun: false
[rp-stat-io-1] ERROR com.epam.reportportal.service.statistics.StatisticsService - Unable to send statistics
java.io.IOException: Unexpected response code for CONNECT: 405
	at okhttp3.internal.connection.RealConnection.createTunnel(RealConnection.java:427)
	at okhttp3.internal.connection.RealConnection.connectTunnel(RealConnection.java:236)
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:177)
	at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.java:224)
	at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.java:108)
	at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.java:88)
	at okhttp3.internal.connection.Transmitter.newExchange(Transmitter.java:169)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:41)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229)
	at okhttp3.RealCall.execute(RealCall.java:81)
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:204)
	at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:46)
	at io.reactivex.Observable.subscribe(Observable.java:12267)
	at io.reactivex.internal.operators.observable.ObservableSubscribeOn$SubscribeTask.run(ObservableSubscribeOn.java:96)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1623)`

Please advice.

**Comment by irynakozak2** [comment]
@zhouhyi Hi, please consider updating the ReportPortal to the latest. I see no such errors in the logs above. However, compatibility issues may happen (WARN message "Unable to complete execution of all dependencies") with the old ReportPortal version. Not sure it is the same issue as the original one.
Please consider creating a new issue if the problem still persists after the ReportPortal version update.


> Our team also encountered the same problem, has anyone solved it?
> 
> @irynakozak2 Can you help check and resolve this issue?
> 
> Environment: com.epam.reportportal: 5.2.1 junit:4.12
> 
> RP is also installed using docker: reportportal/service-auto-analyzer:5.7.4 reportportal/service-auto-analyzer:5.7.4 reportportal/service-api:5.7.4 reportportal/service-jobs:5.7.4 reportportal/service-authorization:5.7.4 reportportal/service-metrics-gatherer:5.7.3 reportportal/service-index:5.7.3 rabbitmq:3.7.16-management traefik:v2.0.7 docker.elastic.co/elasticsearch/elasticsearch:7.10.1 minio/minio:RELEASE.2020-10-27T04-03-55Z postgres:12-alpine reportportal/service-ui:5.7.4
> 
> Running log: 20:19:22.129 [rp-io-28] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID 'f2907150-6076-42ea-bc8e-0d34f4565a46' has been created 20:19:22.129 [rp-io-28] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.129 [rp-io-28] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-28 20:19:22.129 [rp-io-28] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-28 20:19:22.130 [rp-io-28] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-28 20:19:22.131 [rp-io-47] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID 'c31e16d3-53f6-4c93-960e-7e94d62768c0' has been created 20:19:22.131 [rp-io-47] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.131 [rp-io-72] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '079c9e2f-1e0d-4de8-9662-b64b4367ca42' has been created 20:19:22.131 [rp-io-72] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.131 [rp-io-47] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-47 20:19:22.131 [rp-io-47] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-47 20:19:22.147 [rp-io-73] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '3fb23cbe-9aa9-4347-bcf4-71e935c08e78' has been created 20:19:22.147 [rp-io-74] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '1ee79eca-865e-421c-baab-ceefda94324d' has been created 20:19:22.148 [rp-io-73] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.148 [rp-io-74] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.151 [rp-io-90] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '3b30553a-cd64-4665-9e69-a6ba778d9881' has been created 20:19:22.151 [rp-io-93] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '62e92708-b6e3-429c-8a1a-a338c45f6328' has been created 20:19:22.152 [rp-io-90] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.152 [rp-io-93] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.152 [rp-io-93] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-93 20:19:22.152 [rp-io-93] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-93 20:19:22.152 [rp-io-93] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-93 20:19:22.152 [rp-io-93] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-93 20:19:22.153 [rp-io-93] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-93 20:19:22.172 [rp-io-91] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '4fa5602e-2f3c-400d-bab9-53de7f0ae12f' has been created 20:19:22.172 [rp-io-91] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.172 [rp-io-97] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 1ee79eca-865e-421c-baab-ceefda94324d 20:19:22.173 [rp-io-97] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.180 [rp-io-92] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '467682cd-bf99-4bb3-8370-06ddcf2cf3b2' has been created 20:19:22.180 [rp-io-92] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.180 [rp-io-94] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '4084acf6-1635-465a-a5fd-432a068c6050' has been created 20:19:22.180 [rp-io-94] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.180 [rp-io-94] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-94 20:19:22.180 [rp-io-98] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 3fb23cbe-9aa9-4347-bcf4-71e935c08e78 20:19:22.180 [rp-io-94] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-94 20:19:22.180 [rp-io-98] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.181 [rp-io-94] DEBUG com.epam.reportportal.service.Launch - Starting test item...rp-io-94 20:19:22.183 [rp-io-99] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '6cbf2717-f012-4ba0-b5b1-070ea12173ab' has been created 20:19:22.184 [rp-io-99] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.222 [rp-io-91] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 4fa5602e-2f3c-400d-bab9-53de7f0ae12f 20:19:22.222 [rp-io-96] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 079c9e2f-1e0d-4de8-9662-b64b4367ca42 20:19:22.222 [rp-io-4] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID 'db042397-7783-470e-b110-299aa2ef4f41' has been created 20:19:22.222 [rp-io-91] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.222 [rp-io-96] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.222 [rp-io-4] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.229 [rp-io-99] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 6cbf2717-f012-4ba0-b5b1-070ea12173ab 20:19:22.229 [rp-io-99] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.232 [rp-io-93] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID 'dce64ea1-1c59-4949-bd88-756ac72c9244' has been created 20:19:22.232 [rp-io-10] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '75f5a5f7-a5cf-41ea-bcd1-31fabe6c1062' has been created 20:19:22.232 [rp-io-93] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.232 [rp-io-10] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.233 [rp-io-94] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '430609ca-5f88-458b-8a6b-6f161a6d1187' has been created 20:19:22.233 [rp-io-94] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.237 [rp-io-3] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '3096794a-9a3d-4146-8598-d25edca92a2f' has been created 20:19:22.237 [rp-io-3] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.242 [rp-io-9] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID 'dbe97bcc-2269-4296-8d31-93c35e368b6b' has been created 20:19:22.242 [rp-io-9] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.243 [rp-io-1] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 3b30553a-cd64-4665-9e69-a6ba778d9881 20:19:22.243 [rp-io-1] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.246 [rp-io-100] DEBUG com.epam.reportportal.service.Launch - ReportPortal item with ID '122e6452-b1eb-4312-b3fa-4b96de0a287b' has been created 20:19:22.246 [rp-io-100] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Start test item successfully completed 20:19:22.247 [rp-io-92] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 467682cd-bf99-4bb3-8370-06ddcf2cf3b2 20:19:22.247 [rp-io-92] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.255 [rp-io-94] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 430609ca-5f88-458b-8a6b-6f161a6d1187 20:19:22.256 [rp-io-94] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.257 [rp-io-18] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = dce64ea1-1c59-4949-bd88-756ac72c9244 20:19:22.257 [rp-io-18] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.257 [rp-io-19] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 75f5a5f7-a5cf-41ea-bcd1-31fabe6c1062 20:19:22.257 [rp-io-19] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.257 [rp-io-96] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = db042397-7783-470e-b110-299aa2ef4f41 20:19:22.257 [rp-io-96] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.257 [rp-io-4] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = b63f07b7-67fe-49a9-b37f-ace8373aed77 20:19:22.258 [rp-io-4] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.272 [rp-io-24] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 3096794a-9a3d-4146-8598-d25edca92a2f 20:19:22.273 [rp-io-24] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.275 [rp-io-1] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = dbe97bcc-2269-4296-8d31-93c35e368b6b 20:19:22.275 [rp-io-1] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.299 [rp-io-92] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = f2907150-6076-42ea-bc8e-0d34f4565a46 20:19:22.300 [rp-io-92] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.301 [rp-io-30] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 122e6452-b1eb-4312-b3fa-4b96de0a287b 20:19:22.301 [rp-io-30] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.320 [rp-io-1] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 4084acf6-1635-465a-a5fd-432a068c6050 20:19:22.320 [rp-io-1] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.326 [rp-io-92] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 70931c66-31a8-4ab3-8b59-29d7728f689a 20:19:22.326 [rp-io-92] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.327 [rp-io-30] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 62e92708-b6e3-429c-8a1a-a338c45f6328 20:19:22.327 [rp-io-30] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.352 [rp-io-30] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = c31e16d3-53f6-4c93-960e-7e94d62768c0 20:19:22.352 [rp-io-30] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.379 [rp-io-42] DEBUG com.epam.reportportal.service.Launch - Accepted finish request for test item ID = 3277e528-9c51-4b96-a095-1cfff7d18f73 20:19:22.379 [rp-io-42] DEBUG com.epam.reportportal.utils.SubscriptionUtils - Finish test item completed 20:19:22.402 [rp-io-43] DEBUG com.epam.reportportal.service.Launch - null 20:24:22.420 [main] WARN com.epam.reportportal.service.statistics.StatisticsService - Unable to complete execution of all dependencies java.util.concurrent.TimeoutException: The source did not signal an event for 300 seconds and has been terminated. at io.reactivex.internal.operators.completable.CompletableTimeout$DisposeTask.run(CompletableTimeout.java:109) at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38) at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26) at java.util.concurrent.FutureTask.run(FutureTask.java:266) at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) at java.lang.Thread.run(Thread.java:750) [INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 301.2 s -- in TestRunner
> 
> service-API log: 2024-04-01 18:05:16 2024-04-01 10:05:16.464 INFO 1 --- [-listener-exec3] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing started for 1 items. 2024-04-01 18:05:16 2024-04-01 10:05:16.500 INFO 1 --- [-listener-exec3] .l.DirectReplyToMessageListenerContainer : SimpleConsumer [queue=amq.rabbitmq.reply-to, index=0, consumerTag=amq.ctag-vsRqwNyxpqZuVKBsyLhWEg identity=241f698e] started 2024-04-01 18:05:16 2024-04-01 10:05:16.711 INFO 1 --- [-listener-exec3] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing of 1 logs is finished for 1 items. 2024-04-01 18:05:16 2024-04-01 10:05:16.736 INFO 1 --- [-listener-exec3] c.e.t.r.c.a.a.impl.AnalyzerServiceImpl : Start analysis of '1' items for launch with id '36524' 2024-04-01 18:05:16 2024-04-01 10:05:16.884 INFO 1 --- [-listener-exec3] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing started for 1 items. 2024-04-01 18:05:17 2024-04-01 10:05:17.150 INFO 1 --- [-listener-exec3] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing of 1 logs is finished for 1 items. 2024-04-01 18:12:27 2024-04-01 10:12:27.956 INFO 1 --- [-listener-exec4] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing started for 1 items. 2024-04-01 18:12:27 2024-04-01 10:12:27.995 INFO 1 --- [-listener-exec4] .l.DirectReplyToMessageListenerContainer : SimpleConsumer [queue=amq.rabbitmq.reply-to, index=0, consumerTag=amq.ctag-MPgReFl8F3FJO_fhPSr9vQ identity=6ad4872] started 2024-04-01 18:12:28 2024-04-01 10:12:28.148 INFO 1 --- [-listener-exec4] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing of 1 logs is finished for 1 items. 2024-04-01 18:12:29 2024-04-01 10:12:29.633 INFO 1 --- [-listener-exec4] c.e.t.r.c.a.a.impl.AnalyzerServiceImpl : Start analysis of '1' items for launch with id '36525' 2024-04-01 18:12:29 2024-04-01 10:12:29.773 INFO 1 --- [-listener-exec4] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing started for 1 items. 2024-04-01 18:12:29 2024-04-01 10:12:29.943 INFO 1 --- [-listener-exec4] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing of 1 logs is finished for 1 items. 2024-04-01 18:17:11 2024-04-01 10:17:11.272 INFO 1 --- [-listener-exec5] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing started for 1 items. 2024-04-01 18:17:11 2024-04-01 10:17:11.301 INFO 1 --- [-listener-exec5] .l.DirectReplyToMessageListenerContainer : SimpleConsumer [queue=amq.rabbitmq.reply-to, index=0, consumerTag=amq.ctag-bTWfrxyqCv5OWX3OxBaX5g identity=1dfbe391] started 2024-04-01 18:17:11 2024-04-01 10:17:11.554 INFO 1 --- [-listener-exec5] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing of 1 logs is finished for 1 items. 2024-04-01 18:17:12 2024-04-01 10:17:12.106 INFO 1 --- [-listener-exec5] c.e.t.r.c.a.a.impl.AnalyzerServiceImpl : Start analysis of '1' items for launch with id '36526' 2024-04-01 18:17:12 2024-04-01 10:17:12.241 INFO 1 --- [-listener-exec5] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing started for 1 items. 2024-04-01 18:17:12 2024-04-01 10:17:12.419 INFO 1 --- [-listener-exec5] c.e.t.r.c.a.a.indexer.BatchLogIndexer : Indexing of 1 logs is finished for 1 items. 2024-04-01 18:39:41 2024-04-01 10:39:41.102 INFO 1 --- [portal_Worker-1] c.e.t.r.job.InterruptBrokenLaunchesJob : Interrupt broken launches job has been started 2024-04-01 18:39:41 2024-04-01 10:39:41.193 INFO 1 --- [portal_Worker-1] c.e.t.r.job.CleanExpiredCreationBidsJob : Cleaning expired user creation bids finished. Deleted 0 bids 2024-04-01 19:39:41 2024-04-01 11:39:41.095 INFO 1 --- [portal_Worker-1] c.e.t.r.job.InterruptBrokenLaunchesJob : Interrupt broken launches job has been started 2024-04-01 19:39:41 2024-04-01 11:39:41.125 INFO 1 --- [portal_Worker-1] c.e.t.r.job.CleanExpiredCreationBidsJob : Cleaning expired user creation bids finished. Deleted 0 bids

**Comment by irynakozak2** [comment]
> @irynakozak2 I have upgraded TestNG agent version to 5.3.1 and I still see 405 error. Here is the service-API log `2023-08-24 18:54:23.977 ERROR 1 --- [585-exec-160036] .AuthUtils$SerialUidReplacingInputStream : Potentially Fatal Deserialization Operation.
> 
> java.io.InvalidClassException: Overriding serialized class version mismatch: local serialVersionUID = 550 stream serialVersionUID = 520 at com.epam.ta.reportportal.auth.util.AuthUtils$SerialUidReplacingInputStream.readClassDescriptor(AuthUtils.java:105) at java.base/java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1992) at java.base/java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1870) at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2201) at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687) at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:489) at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:447) at java.base/java.util.TreeSet.readObject(TreeSet.java:524) at java.base/jdk.internal.reflect.GeneratedMethodAccessor2304.invoke(Unknown Source) at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) at java.base/java.lang.reflect.Method.invoke(Method.java:566) at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:1046) at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2357) at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228) at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687) at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496) at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390) at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228) at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687) at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496) at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390) at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228) at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687) at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496) at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390) at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228) at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687) at java.base/java.io.ObjectInputStream.defaultReadFields(ObjectInputStream.java:2496) at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2390) at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2228) at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1687) at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:489) at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:447) at com.epam.ta.reportportal.auth.util.AuthUtils.deserializeSafely(AuthUtils.java:69) at com.epam.ta.reportportal.auth.CombinedTokenStore.readAuthentication(CombinedTokenStore.java:68) at org.springframework.security.oauth2.provider.token.store.JwtTokenStore.readAuthentication(JwtTokenStore.java:70) at com.epam.ta.reportportal.auth.CombinedTokenStore.readAuthentication(CombinedTokenStore.java:55) at com.epam.ta.reportportal.auth.CombinedTokenStore$$FastClassBySpringCGLIB$$753697c.invoke() at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:783) at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:753) at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:123) at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:388) at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:753) at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:698) at com.epam.ta.reportportal.auth.CombinedTokenStore$$EnhancerBySpringCGLIB$$7c24aad6.readAuthentication() at org.springframework.security.oauth2.provider.token.DefaultTokenServices.loadAuthentication(DefaultTokenServices.java:242) at org.springframework.security.oauth2.provider.token.DefaultTokenServices$$FastClassBySpringCGLIB$$5a1f25c.invoke() at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:689) at org.springframework.security.oauth2.provider.token.DefaultTokenServices$$EnhancerBySpringCGLIB$$a07cd176.loadAuthentication() at org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationManager.authenticate(OAuth2AuthenticationManager.java:87) at org.springframework.security.oauth2.provider.authentication.OAuth2AuthenticationProcessingFilter.doFilter(OAuth2AuthenticationProcessingFilter.java:154) at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336) at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:103) at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:89) at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336) at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90) at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75) at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336) at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:110) at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:80) at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336) at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:55) at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:336) at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:211) at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:183) at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:354) at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:267) at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93) at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.doFilterInternal(WebMvcMetricsFilter.java:96) at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201) at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202) at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143) at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346) at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374) at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887) at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684) at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) at java.base/java.lang.Thread.run(Thread.java:829) `
> 
> run log `[main] INFO com.epam.reportportal.service.Launch - Rerun: false [rp-stat-io-1] ERROR com.epam.reportportal.service.statistics.StatisticsService - Unable to send statistics java.io.IOException: Unexpected response code for CONNECT: 405 at okhttp3.internal.connection.RealConnection.createTunnel(RealConnection.java:427) at okhttp3.internal.connection.RealConnection.connectTunnel(RealConnection.java:236) at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:177) at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.java:224) at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.java:108) at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.java:88) at okhttp3.internal.connection.Transmitter.newExchange(Transmitter.java:169) at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:41) at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142) at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117) at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94) at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142) at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117) at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93) at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142) at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88) at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142) at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117) at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229) at okhttp3.RealCall.execute(RealCall.java:81) at retrofit2.OkHttpCall.execute(OkHttpCall.java:204) at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:46) at io.reactivex.Observable.subscribe(Observable.java:12267) at io.reactivex.internal.operators.observable.ObservableSubscribeOn$SubscribeTask.run(ObservableSubscribeOn.java:96) at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38) at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26) at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317) at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) at java.base/java.lang.Thread.run(Thread.java:1623)`
> 
> Please advice.

@mchundi Hi, please also update the ReportPortal version at least to [23.2](https://github.com/reportportal/reportportal/releases/tag/v23.2), we've addressed the deserialization issue there.
